### ISSUE TITLE
tech/aria/heading-role-no-level nvda/chrome

### ISSUE THREAD

**Issue by SimoAccessiWay** [accessibilitysupported/a11ysupport.io#301]
This Support Point submission is for the test [ARIA heading role with no aria-level attribute](/tests/tech__aria__heading-role-no-level)

meta info

| property | value |
| --- | --- |
| title | tech/aria/heading-role-no-level |
| at | nvda |
| at_version | 2024.3 |
| browser | chrome |
| browser_version | 129 |
| os_version | Windows 11 version 23H2 |

## Test case: Use NVDA + F7 (List headings) to navigate forward to `*[role="heading"]`
Command used: `list_headings`

### output
**before:**
```nested under the h1```
**after:**
```Level 0 aria heading with missing aria-level attribute, expanded 1 of 1```
